sdk/src/test/java/com/stormpath/sdk/StormpathConfigurationTest.java

```diff
@@ -0,0 +1,45 @@
+package com.stormpath.sdk;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StormpathConfigurationTest extends BaseTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void exceptionThrownIfBaseUrlNotSet() throws Exception {
+        new StormpathConfiguration.Builder().build();
+    }
+
+    @Test
+    public void pathsStartWithSlash() throws Exception {
+        StormpathConfiguration stormpath = new StormpathConfiguration.Builder()
+                .baseUrl("https://api.stormpath.com/v1")
+                .oauthPath("my-oauth")
+                .registerPath("my-register")
+                .passwordResetPath("my-password-reset")
+                .verifyEmailPath("my-verify-email")
+                .userProfilePath("my-me-path")
+                .logoutPath("my-logout-path")
+                .build();
+
+        assertThat(stormpath.oauthPath()).isEqualTo("/my-oauth");
+        assertThat(stormpath.registerPath()).isEqualTo("/my-register");
+        assertThat(stormpath.passwordResetPath()).isEqualTo("/my-password-reset");
+        assertThat(stormpath.verifyEmailPath()).isEqualTo("/my-verify-email");
+        assertThat(stormpath.userProfilePath()).isEqualTo("/my-me-path");
+        assertThat(stormpath.logoutPath()).isEqualTo("/my-logout-path");
+    }
+
+    @Test
+    public void baseUrlWithoutTrailingSlash() throws Exception {
+        StormpathConfiguration stormpath = new StormpathConfiguration.Builder().baseUrl("https://example.com/").build();
```

Comment:
Actually, I take it back - in the context of the Android SDK, it will be hitting an integration server - like, a running express server with the Stormpath integration.